Uplift/Views/Onboarding/CheckBoxView.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  CheckBoxView.swift
+//  Uplift
+//
+//  Created by Belle Hu on 9/22/24.
+//  Copyright © 2024 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct CheckBoxView: View {
+
+    // MARK: - Properties
+
+    @Binding var isCheckedTerms: Bool
+    @Binding var isCheckedData: Bool
+    @Binding var isCheckedLocation: Bool
```

Comment:
Nit: Usually the convention for boolean values is to use "did____"

EX: didCheckTerms, didCheckData, didCheckLocation